kuma/static/latin-subset

```diff
@@ -0,0 +1,17 @@
+# Basic Latin
+U+0020-007F
+
+# Latin-1 Supplement
+U+00A0-00FF
+
+# Latin Extended-A
+U+0100-017F
+
+# Latin Extended-B
+U+0180-024F
+
+# General Punctuation
+U+2000-206F
+
+# Currency Symbols
+U+20A0-20CF
```

Comment:
Nit: Missing empty line at end of file. This seems to be a new file? I am genuinely curious as to the purpose of this file? @tkadlec ?

Reply:
Added the new line. Also added some info to the top of the file for clarity. The file is used to generate the subset. You can see more information here: https://github.com/mdn/sprints/issues/254#issuecomment-427059741